CONTRIBUTING.md

```diff
@@ -0,0 +1,297 @@
+# Contributing to OPM Simulators
+
+Thank you for your interest in contributing to the Open Porous Media (OPM) Simulators project! This guide will help you get started with contributing code, documentation, or bug reports.
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [Development Process](#development-process)
+- [Code Style and Formatting](#code-style-and-formatting)
+- [Automated Formatting Tools](#automated-formatting-tools)
+- [Testing](#testing)
+- [Submitting Changes](#submitting-changes)
+- [Reporting Issues](#reporting-issues)
+- [Getting Help](#getting-help)
+
+## Getting Started
+
+Before contributing, please:
+
+1. Read the [OPM Project Documentation](https://opm-project.org)
+2. Review the [OPM Manual](https://opm-project.org/?page_id=955)
+3. Ensure you have the necessary [prerequisites](https://opm-project.org/?page_id=239) installed
+
+## Development Process
+
+### Fork and Clone
+
+1. Fork the repository on GitHub
+2. Clone your fork locally:
+   ```bash
+   git clone https://github.com/YOUR-USERNAME/opm-simulators.git
+   cd opm-simulators
+   ```
+
+3. Add the upstream repository:
+   ```bash
+   git remote add upstream https://github.com/OPM/opm-simulators.git
+   ```
```

Comment:
I would actually do it the other way around (clone OPM and then add the user fork)

Like this master will track the fork and people might already have a check out of the original repo. You can rename remotes rather easily.

Reply:
I would prefer the other way around, but that is a matter of taste, so I do not request any change, considering this resolved for now.